metadata.yaml

```diff
@@ -34,6 +34,9 @@ storage:
   mongodb:
     type: filesystem
     location: /var/snap/charmed-mongodb/common
+  logs:
+    type: filesystem
```

Comment:
is juju refresh possible to version with changed storages list?

Reply:
@delgod, indeed juju will not accept that. It runs a mountpoint validation [here](https://github.com/juju/juju/blob/1c4005149780c7d359c9b3a38c4b0371f9eeb1ed/state/filesystem.go#L1476-L1481). If one mount point contains the other, it will fail.

The only alternative is to update the `mongodb` mount point to a subfolder that does not contain logs's. However, that will be a breaking change between revisions. The upper side is keeping IOPS of logging and data separated.

Reply:
Removed the mount point for now